kamelet-support/src/main/java/io/kaoto/backend/model/deployment/kamelet/Bean.java

```diff
@@ -29,7 +29,7 @@
  value: 'org.postgresql.Driver'
  * ```
  */
-@JsonPropertyOrder({"name", "type", "property"  })
+@JsonPropertyOrder({"name", "type", "properties", "property"  })
```

Comment:
If we are going to convert all to property instead of properties, maybe it makes sense not to mention it here?

Reply:
Aren't we converting to `properties`? Although both of them will stay supported, the recommended way is `properties` which is same with Camel Route/Integration beans.

Reply:
Uh... maybe I was remembering right. Yes.